includes/pet-details-form.php

```diff
@@ -169,3 +170,5 @@
         </svg>
     </span>
 </p>
+<button>Submit</button>
+</form>
```

Comment:
Also, only added this so I could test the form and put some data in the database - should allow you to create more detailed cards